src/plugins/workspace/server/plugin.ts

```diff
@@ -140,6 +142,7 @@ export class WorkspacePlugin implements Plugin<{}, {}> {
           name: i18n.translate('workspaces.management.workspace.default.name', {
             defaultMessage: 'Management',
           }),
+          features: [`@${DEFAULT_APP_CATEGORIES.management.id}`],
```

Comment:
Should we add features when creating public workspace?

Reply:
Good question, do we have a conclusion already of what features to show in a `public` workspace?

Reply:
I think one thing for sure is that management category should not be displayed in public workspace.